docker-compose.yml

```diff
@@ -92,6 +92,7 @@ services:
       OIDC_CLIENT_SECRET: ${DATASERVICES_GOB_EXPORT_CLIENT_SECRET}
       AIRFLOW_LOG_CLEANUP__MAX_LOG_AGE_IN_DAYS: 90
       AIRFLOW_LOG_CLEANUP__ENABLE_DELETE_CHILD_LOG: "true"
+      DSO_DB_POSTGRES_PASSWORD: insecure
```

Comment:
Dit is deel van `AIRFLOW_CONN_POSTGRES_DEFAULT` en al beschikbaar.

Reply:
Het bijzonder is aan de pg_comparator dat alleen een env var wordt geaccepteerd en geen pickle uit een bestaande voor het wachtwoord. Tooltje is een beetje spartaans in dat opzicht...

Reply:
Maar deze env variabel is nodig voor de subprocess call, niet voor de airflow zelf, toch?

Reply:
Ja klopt, voor de subproces. Alleen deze aanmaken in runtime lukte niet (export envar=tekst)

Reply:
`subprocess.run` accepteerd `env` parameter (dict) waar je kan alles zetten:
https://docs.python.org/3/library/subprocess.html#subprocess.run

```
subprocess.run(..., env=dict(DSO_DB_POSTGRES_PASSWORD="meh"))
```

Reply:
Yeay! gelukt, eindelijk